### PR TITLE
Update GKE test grid with 1.6 and 1.7 branch tests.

### PR DESCRIPTION
**Pull request by fabioy**
This is to fix issue #3303.

### CODE REVIEW

**Comment by krzyzacy** [comment]
/lgtm

seems your cla is screwed

File: testgrid/config/config.yaml [hunk 1]
```diff
@@ -1511,12 +1511,11 @@ dashboards:
   dashboard_tab:
   - name: gke
     test_group_name: ci-kubernetes-e2e-gke
+  # K8S head on GKE
```
Comment (krzyzacy) [inline testgrid/config/config.yaml]: small nit: this comment probably need to be before the above gke job.

Comment (fabioy) [inline testgrid/config/config.yaml]: Removed. It wasn't needed.

**Comment by fabioy** [comment]
Thanks, fixed CLA. Had my email address wrong in the PR.

**Pull request merged**